src/usage_metrics/models.py

```diff
@@ -507,3 +508,598 @@
     Column("remote_ip_full_location", String),
     Column("partition_key", String),
 )
+
+
+core_eel_hole_logs = Table(
```

Comment:
non-blocking.... ah okay you kept this entire wide table and then broke up the bits? it seems like this table could be an asset but not a db table. but also how does the fact that the asset is named _core_eel_hole_logs work?